salt/modules/ansiblegate.py

```diff
@@ -147,6 +147,10 @@ def call(self, module, *args, **kwargs):
         :param kwargs: keywords to the module
         :return:
         '''
+        if six.PY3:
+            python_exec = 'python3'
+        else:
+            python_exec = 'python'
```

Comment:
In Arch Linux, `python` maps to `python3` and `python2` needs to be explicitly specified.   This code will probably be fine, I just wanted to leave a note in case someone on Arch Linux opens up an issue later.

Reply:
I agree we might not want to leave this static. Can we make this a kwarg instead and only set if the kwarg is not set?